pdudaemon/drivers/devantech.py

```diff
@@ -60,11 +61,13 @@ def port_interaction(self, command, port_number):
         if port_number > self.port_count:
             log.error("There are only %d ports. Provide a port number lesser than %d." % (self.port_count, self.port_count))
             raise RuntimeError("There are only %d ports. Provide a port number lesser than %d." % (self.port_count, self.port_count))
-
+        if self.logic not in ["NO", "NC"]:
```

Comment:
would it be worth to convert to lowercase & check the outcome of that ?
There are other devantech modules which can have other logic levels (e.g. active-low, active-low-pullup, open-collector) so it may also be worth accepting "no" / "NO"  (& "normally-open" in future).

Reply:
I think this could be added in a separate PR when needed.

Reply:
Sounds perfect to me. I don't have the hardware anyway as yet so no need to block here ;-)